backend/services/api.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
```

Comment:
If you want you can store the Poke API root URL at some global variable, so you can access easily at other scopes. Again it isn't wrong the way you did, feel free to choose what you prefer :)

```suggestion
POKEAPI_ROOT_URL = 'https://pokeapi.co/api/v2/pokemon/'
```